tests/console_test.go

```diff
@@ -38,25 +38,40 @@ var _ = Describe("Console", func() {
 
 	flag.Parse()
 
-	virtClient, err := kubecli.GetKubevirtClient()
+	var virtClient kubecli.KubevirtClient
+	var err error
+
 	tests.PanicOnError(err)
 
 	BeforeEach(func() {
 		tests.BeforeTestCleanup()
 	})
 
-	RunVMIAndExpectConsoleOutput := func(vmi *v1.VirtualMachineInstance, expected string) {
+	JustBeforeEach(func() {
+		By("Opening new virtClient")
+		virtClient, err = kubecli.GetKubevirtClient()
```

Comment:
Is this change also related to the expecter?

Reply:
@rmohr thanks, this change is related to the second PR I have in progress. This one is the concept I tested. Removing it now.